### DISCUSSION TITLE
essay/2024/05/20/app-vs-web

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#212] [Comments]
# essay/2024/05/20/app-vs-web

서비스를 웹으로 만들지 앱으로 만들지 고민하던 날들이 생각납니다. 각각 장단점이 있으니까.

https://jeho.page/essay/2024/05/20/app-vs-web.html

**Comment by bookseal** [comment]
웹이냐 앱이냐로 고민하는 사람이 많은 만큼 flutter가 이제 트랜드가 됐네요
